docs/contributor/04-10-technical-design.md

```diff
@@ -38,3 +38,28 @@ In the event of a failure during the reconciliation, APIRule Controller performs
 The following diagram illustrates the reconciliation process of APIRule and the created resources:
 
 ![APIRule CR Reconciliation](../assets/api-rule-reconciliation-sequence.svg)
+
+## Labeling resources
```

Comment:
Don't know, it could be nice to have documented some specific of our reconciliation and what it does. I might remove the `kubectl get` commands, but keep the labels we are setting. Each feature we introduced it will be nice to have corresponding documentation with it.

Reply:
You prefer no doc in this case?

Reply:
Sorry, to be unclear. Right now this is only a contributor documentation, but I think it should be user-facing.